packages/block-library/src/index.js

```diff
@@ -21,37 +21,39 @@ import {
 //
 // See https://github.com/WordPress/gutenberg/pull/40655 for more context.
 import * as archives from './archives';
-import * as avatar from './avatar';
 import * as audio from './audio';
+import * as avatar from './avatar';
+import * as reusableBlock from './block';
```

Comment:
Some of the changes in the this file seem unintentional, can you revert those changes please and only include the new import. It looks like these belong in alphabetical order.